### PR TITLE
Add import globbing docs

### PR DESCRIPTION
**Pull request by matkuliak**
## Summary of the changes / Why this is an improvement

Added brief globbing docs

## Checklist

 - [x] Link to issue this PR refers to (if applicable): https://github.com/crate/cloud/issues/1505 / 
 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: docs/reference/overview.rst [hunk 1]
```diff
@@ -322,10 +322,10 @@ Import from private S3 bucket
 
 CrateDB Cloud allows convenient imports directly from S3-compatible storage. 
 To import a file form bucket, provide the name of your bucket, and path to
-the file. The S3 Access Key ID, and S3 Secret Access Key are also needed. You can
-also specify the endpoint for non-AWS S3 buckets. Keep in mind that you may be
-charged for egress, depending on your provider. There is also a limit of 10 GiB
-for S3 imports. The usual file formats are supported.
+the file. The S3 Access Key ID, and S3 Secret Access Key are also needed. You 
+can also specify the endpoint for non-AWS S3 buckets. Keep in mind that you may
+be charged for egress, depending on your provider. There is also a limit of 10
```
Comment (amotl) [inline docs/reference/overview.rst:327]:
```suggestion
be charged for egress traffic, depending on your provider. There is also a volume limit of 10
```

File: docs/reference/overview.rst [hunk 2]
```diff
@@ -322,10 +322,10 @@ Import from private S3 bucket
...
+GiB for S3 imports. The usual file formats are supported.
```
Comment (amotl) [inline docs/reference/overview.rst:328]: Is it "per-file", actually?
```suggestion
GiB per file for S3 imports. The usual file formats are supported.
```

Comment (amotl) [inline docs/reference/overview.rst:328]: > The usual file formats are supported.

Which ones?

File: docs/reference/overview.rst [hunk 3]
```diff
@@ -350,6 +350,24 @@ for S3 imports. The usual file formats are supported.
         }]
         }
 
+.. _overview-cluster-import-globbing:
+
+Import globbing
+~~~~~~~~~~~~~~~
```
Comment (amotl) [inline docs/reference/overview.rst:356]: Globbing is a pretty technical term. Maybe use "wildcard" instead?

File: docs/reference/overview.rst [hunk 4]
```diff
@@ -350,6 +350,24 @@ for S3 imports. The usual file formats are supported.
...
+
```
Comment (amotl) [inline docs/reference/overview.rst:357]: What about such a little introductory paragraph before going into the technical details?

> You can import multiple files from an S3 bucket at once by using the well-known
> wildcard notation, also known as "globbing". In computer programming, [glob patterns]
> specify sets of filenames with [wildcard characters].

[glob patterns]: https://en.wikipedia.org/wiki/Glob_(programming
[wildcard characters]: https://en.wikipedia.org/wiki/Wildcard_characters

**Review by amotl — CHANGES_REQUESTED** [review]
Thanks for improving that section. I've added a few suggestions, feel free to use them or not.

File: docs/reference/overview.rst [hunk 5]
```diff
@@ -350,6 +350,24 @@ for S3 imports. The usual file formats are supported.
         }]
         }
 
+.. _overview-cluster-import-globbing:
+
+Import globbing
```
Comment (proddata) [inline docs/reference/overview.rst:355]:
```suggestion
Importing multiple files
```

**Comment by proddata** [comment]
Can we adjust this PR to also reflect the changes of https://github.com/crate/cloud/issues/1505 (Azure Import).
This is similiar to S3 Imports and also supports multi file imports

**Comment by matkuliak** [comment]
Added section on imports for Azure Container/Blob Storage and also mentioned it in multiple-files-import section.

File: docs/reference/overview.rst [hunk 6]
```diff
@@ -350,6 +350,55 @@ for S3 imports. The usual file formats are supported.
         }]
         }
 
+.. _overview-cluster-import-azure:
+
+Azure Container Storage
+^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (proddata) [inline docs/reference/overview.rst:356]:
```suggestion
Import from Azure Blob Storage Container
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

File: docs/reference/overview.rst [hunk 7]
```diff
@@ -325,7 +325,7 @@ the file. The S3 Access Key ID, and S3 Secret Access Key are also needed. You
 can also specify the endpoint for non-AWS S3 buckets. Keep in mind that you may
 be charged for egress traffic, depending on your provider. There is also a
 volume limit of 10 GiB per file for S3 imports. The usual file formats are
-supported.
+supported - CSV, Json, and Parquet.
```
Comment (SStorm) [inline docs/reference/overview.rst]:
```suggestion
supported - CSV (all variants), Json (JSON-Lines, JSON Arrays and JSON Documents), and Parquet.
```

File: docs/reference/overview.rst [hunk 8]
```diff
@@ -350,6 +350,55 @@ for S3 imports. The usual file formats are supported.
...
+Import from Azure Blob Storage Container
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Importing from Azure Storage Containers is also supported. In this case, the
+secret consists of a secret name, an Azure Storage Connection string or an
+Azure SAS Token URL.
+
+There is an option to import data from a private Azure Blob Storage container
+using a stored secret. Secret can be added by an admin user at the organization
+level.
```
Comment (proddata) [inline docs/reference/overview.rst:364]: this seems to be a duplicate. maybe something like this ?

```suggestion
Importing data from private Azure Blob Storage containers is possible using a stored secret, which includes a secret name and either an Azure Storage Connection string or an Azure SAS Token URL. An admin user at the organization level can add this secret.
```

**Review by proddata — APPROVED** [review]

**Review by amotl — APPROVED** [review]
Thank you 👍. Maybe squash the commits before merging?

**Pull request merged**